dev/ci/ci-color.sh

```diff
@@ -7,7 +7,9 @@ ci_dir="$(dirname "$0")"
 
 git_download color
 
-export COQEXTRAFLAGS='-native-compiler no'
+ulimit -s
+ulimit -s 65536
+ulimit -s
```

Comment:
This is just checking the stack size before and after setting it right? Can it be expected to not work?

Reply:
> This is just checking the stack size before and after setting it right?

Correct

> Can it be expected to not work?

Well, I just thought it would be nice to get some log, just in case.